### ISSUE TITLE
Blocking webhost IP address?

### ISSUE THREAD

**Issue by n0ttsweet** [Yelp/yelp-fusion#236]
I have been trying to troubleshoot an issue I am having with a Yelp Reviews plugin I purchased. All symptoms seem to point at the culprit being my IP address. 
I'm very much a rookie, but is there any way I can test and see if my webhost's IP was in fact blocked, and if so, why?
I could go purchase a unique IP, but if the plugin is causing some violation of your Terms, I'd like to avoid that altogether.

**Comment by mittonk** [comment]
@n0ttsweet : Do you have a link to the plugin you purchased?

**Comment by n0ttsweet** [comment]
Yes sir!
https://codecanyon.net/item/yelp-reviews-pro-for-wordpress/15376445

For the record, we have done some testing. Two websites sharing the same IP do not work.
One is : pole.sweetsolutionsllc.com the other is pole.cheskitchen.com
I purchased a different plugin, so pole.sweetsolutionsllc.com is now working, but I'd still like to resolve the issue. 

Additionally, I thought perhaps my API key was bugged, so I went to a different hosting plan with a different IP and tested it at test.maurersales.com 

Any help or even clarification would be fantastic... The IP in question is : 208.113.221.129

Thank you!!
